libvast/vast/system/importer.hpp

```diff
@@ -84,6 +86,8 @@ struct importer_state {
                         caf::broadcast_downstream_manager<table_slice>>
     stage;
 
+  transformer_actor transformer;
```

Comment:
This also doesn't need the full transformer interface but just the `stream_sink<table_slice>`